android/gradle.properties

```diff
@@ -2,3 +2,4 @@ org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
 android.enableR8=true
+android.jetifier.blacklist=bcprov-jdk15on
```

Comment:
what is it?

Reply:
It's in the commit message: Added Jetifier blacklist so flutter can build APK
(See these issues: https://github.com/bcgit/bc-java/issues/762 and https://issuetracker.google.com/issues/172784894)